src/Symfony/Component/HttpFoundation/Session/Storage/Handler/StrictSessionHandler.php

```diff
@@ -28,6 +34,15 @@ public function __construct(\SessionHandlerInterface $handler)
         }
 
         $this->handler = $handler;
+        $this->wrapper = $handler instanceof \SessionHandler;
+    }
+
+    /**
+     * Returns true if this handler wraps an internal PHP session save handler using \SessionHandler.
+     */
+    public function isWrapper(): bool
+    {
+        return $this->wrapper;
```

Comment:
```suggestion
        return $this->handler instanceof \SessionHandler;
```

We don't need that additional property, do we?